src/ToDo.js

```diff
@@ -1,21 +1,29 @@
 import React from "react";
 
 const ToDo = ({ todo, handleToggle }) => {
+  const [checked, setChecked] = React.useState(todo.complete);
   const handleClick = (e) => {
+    setChecked(e.currentTarget.checked);
     e.preventDefault();
     handleToggle(e.currentTarget.id);
   };
 
+  const handleChange = (event) => {
+    setChecked(event.target.checked);
+  };
+
   return (
     <div
-      id={todo.id}
+      value={todo.id}
       key={todo.id + todo.task}
+      id={todo.id}
       name="todo"
-      value={todo.id}
       onClick={handleClick}
-      className={todo.complete ? "todo strike" : "todo"}
     >
-      {todo.task}
+      <input type="checkbox" checked={checked} onChange={handleChange} />
```

Comment:
```suggestion
      <input type="checkbox" checked={todo.complete} onChange={handleChange} />
```

Same logic as this line 
https://github.com/SanjanaPurma/sample-react-proj/blob/b99526d9082d8473fd21f418c18fe46669377ee5/src/ToDo.js#L24

Reply:
I tried this way in the beginning Kent, but the **checked** attribute is not getting changed this way.

Reply:
Your onChange function need to call `handleClick` function and need to assign the same id to the checkbox.

Reply:
For example, just a concept, I never test the code.

```
<input type="checkbox" checked={todo.complete} onChange={handleClick} id={todo.id} />
```